components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -578,7 +565,13 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
                                     menuEntries={[
                                         {
                                             title: "Open in Browser",
-                                            onClick: () => window.parent.postMessage({ type: "openBrowserIde" }, "*"),
+                                            onClick: () => {
+                                                // TODO(ak): delete after supervisor deploy
```

Comment:
Is this TODO intended to be left in place?

Reply:
after both deployments, we will do another PR to clean up